mailpoet/assets/js/src/automation/listing/components/cells/actions.tsx

```diff
@@ -21,6 +22,7 @@ export function Actions({ automation }: Props): JSX.Element {
 
   return (
     <div className="mailpoet-automation-listing-cell-actions">
+      <Analytics automation={automation} />
```

Comment:
Same as above: I think this should be filterable and the integration adds the Analytics link.

But imo, we can do this in a separate maintenance ticket.